CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/GiniConfiguration.swift

```diff
@@ -717,4 +718,31 @@ import UIKit
     @objc public var backToMenuButtonResource: PreferredButtonResource?
     @objc public var nextButtonResource: PreferredButtonResource?
     @objc public var cancelButtonResource: PreferredButtonResource?
+    
+    /**
+     Set dictionary of fonts for available text styles. Used internally.
+     */
+    public var textStyleFonts: [UIFont.TextStyle: UIFont] = [
```

Comment:
In Bank SDK's config this is private and here it is public. 🤔 I know that this makes it convenient to pass the font configuration from Bank SDK to Capture SDK, but it exposes internal concerns.

I would recommend to make this private so that the Capture SDK's public API only allows font configuration via `updateFont()`. Then in `GiniBankConfiguration` call `updateFont()` for each text style. It's much more verbose, but it keeps the public API clean.

Reply:
I was trying to do so but I was not able to iterate through UIFont.TextStyle 🤔. I'll check if I can do something else with it.